webtask-templates/visualstudio.json

```diff
@@ -53,7 +53,7 @@
         "TYPE": "git"
       }
     },
-    "PATH": {
+    "PROJECT_PATH": {
```

Comment:
Why did this change?

Reply:
Seems like this secret was broken for quite some time. It is `PROJECT_PATH` in the code. I assume, `tfvc`x isn't very popular thing. And extension doesn't use this secret for `git`.